### PR TITLE
Reorg Ledger Page & Addition DevApp Installation Guidelines

### LINKED ISSUE #3989: Update info Installation Dev version DOT KSM Ledger App
Add the following [this wiki page](https://wiki.polkadot.network/docs/kusama-ledger)

Installation `ledgerblue` required

`python3 -m pip install ledgerblue`

Check if you have latest `pip` version

Then go to the folder with the .sh file of the app and type

`./installer_nanos_plus.sh load`

- Uninstall current App
- Install Dev App
- Insert PIN code to approve

Maybe make a short video tutorial too. It is not recommended to install dev apps but it might be that zondax releases the app and Ledger takes some time to review it and release it on Ledger Live. This lag might leave users with an old version of the app while the UI is up-to-data. As a consequence it would not be pèossible to use the device --> Error message "txn version not supported"

### PR DESCRIPTION
**Pull request by filippoweb3**
See related Issue

### CODE REVIEW

**Comment by filippoweb3** [comment]
hey @emresurmeli let me know what you think about this PR ;)

File: docs/general/ledger.md
Comment (profradha) [inline docs/general/ledger.md]:
```suggestion
Ledger does not currently support batch calls; As a consequence, if you stake using the
[staking dashboard](https://staking.polkadot.network/#/overview), you must use a hot wallet.
Alternatively, you can use the [Polkadot-JS UI](https://polkadot.js.org/apps/#/explorer) that allows
for single calls (i.e., bonding and nominating).
```

File: docs/general/ledger.md
Comment (profradha) [inline docs/general/ledger.md]:
```suggestion
This section is for developers only. It is recommended to install the
```

File: docs/general/ledger.md
Comment (profradha) [inline docs/general/ledger.md]: Zondax may update the apps even when there is no runtime upgrade on the network (like some of the backlogged feature requests). Rephrased the content a bit to reflect that.
Please accept this suggestion if the rephrased content captures the original intent.

```suggestion
Ledger apps for the Polkadot and Kusama ecosystems are developed by [Zondax](https://zondax.ch/).
When new functionalities are added to the Ledger apps, they are made available on a developer release for testing purposes. After a successful audit and review, the apps would be available for download and
installation using [Ledger Live](https://www.ledger.com/ledger-live). As it takes some
time for Ledger to audit and review the release, the app upgrade option may not be 
available on Ledger Live when the new runtime is deployed on the network. If this happens, users cannot use Ledger
devices with the Polkadot-JS UI, and while signing for a transaction, they will most likely
incur the error message "txn version not supported". Please do not panic if this happens, as there
are solutions to this problem. If you cannot wait a couple of days until the app passes the Ledger
audit, you can install the developer release from the shell using the latest version published on
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Looks good. Some minor suggestions.

**Comment by filippoweb3** [comment]
@DrW3RK thanks for the review! When you approve it, please do not merge as I will add a short video showing how to install the dev version of the app. I will do a final check before merging.

Also, Juan at Zondax told me they fixed the batch call issue while nominating with the new release. So, it will be possible to use Ledger with batch call bond+nominate. I will edit the text accordingly.

**Comment by DrW3RK** [comment]
Got it! I'll review again after you add the mentioned contents

**Comment by filippoweb3** [comment]
@DrW3RK this is good to go. Tutorial added. Regarding the batch transfer calls implementation, I will create an issue to update all relevant pages in the wiki.

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**
